scripts/us_bea/states_gdp/import_industry_data_and_gen_mcf.py

```diff
@@ -0,0 +1,168 @@
+ # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pulls data from the US Bureau of Economic Analysis (BEA) on quarterly GDP
+per US state. Saves output as a CSV file.
+
+    Typical usage:
+
+    python3 import_data.py
+"""
+from urllib.request import urlopen
+from absl import app
+import pandas as pd
+import io
+import zipfile
+import csv
+import re
+
+# Suppress annoying pandas DF copy warnings.
+pd.options.mode.chained_assignment = None # default='warn'
+
+# Note: this list will be imported from the US State GDP data file as soon
+# as that is merged.
+US_STATES = ['Alabama', 'Alaska', 'Arizona', 'Arkansas',
+             'California', 'Colorado', 'Connecticut', 'Delaware',
+             'District of Columbia', 'Florida', 'Georgia', 'Hawaii', 'Idaho',
+             'Illinois', 'Indiana', 'Iowa', 'Kansas', 'Kentucky', 'Louisiana',
+             'Maine', 'Maryland', 'Massachusetts', 'Michigan', 'Minnesota',
+             'Mississippi', 'Missouri', 'Montana', 'Nebraska', 'Nevada',
+             'New Hampshire', 'New Jersey', 'New Mexico', 'New York',
+             'North Carolina', 'North Dakota', 'Ohio', 'Oklahoma', 'Oregon',
+             'Pennsylvania', 'Rhode Island', 'South Carolina', 'South Dakota',
+             'Tennessee', 'Texas', 'Utah', 'Vermont', 'Virginia', 'Washington',
+             'West Virginia', 'Wisconsin', 'Wyoming']
+
+# It would be great to make this class inherit from StateGDPDataLoader to
+# avoid repeating e.g. the dowload_data function. As soon as that gets merged
+# in, I will modify this to inherit from there.
+class StateGDPIndustryDataLoader:
+    """Pulls GDP industry-level state data from BEA.
+
+    Attributes:
+        df: DataFrame (DF) with the cleaned data.
+    """
+    ZIP_LINK = "https://apps.bea.gov/regional/zip/SQGDP.zip"
```

Comment:
Should we make these private by prepending underscores?

Reply:
Done